.github/workflows/linux-eic-shell.yml

```
@@ -54,6 +60,14 @@ jobs:
             CXX: clang++
             CMAKE_BUILD_TYPE: Debug
             CXXFLAGS: -fprofile-instr-generate -fcoverage-mapping
+          - CC: clang
+            CXX: clang++
+            CMAKE_BUILD_TYPE: Release
+            release: nightly
+          - CC: clang
+            CXX: clang++
+            CMAKE_BUILD_TYPE: Release
+            release: 23.12-alpha
```

Better approach is to have the fixed set of test releases, and then `include` the `env.release` as well assuming it will get de-duplicated away:
```suggestion
          - CC: clang
            CXX: clang++
            CMAKE_BUILD_TYPE: Release
            release: nightly
          - CC: clang
            CXX: clang++
            CMAKE_BUILD_TYPE: Release
            release: 23.12-alpha
          - CC: clang
            CXX: clang++
            CMAKE_BUILD_TYPE: Release
            release: ${{ env.release }}
```

Then all platform-release strings can become:
```diff
-        platform-release: "${{ env.platform }}:${{ matrix.release || env.release }}"
+        platform-release: "${{ env.platform }}:${{ matrix.release }}"
```

---

... assuming de-duplication works.